clang/test/CodeGenSYCL/loop_unroll.cpp

```diff
@@ -30,7 +30,7 @@ void disable() {
 }
 
 template <typename name, typename Func>
-__attribute__((sycl_kernel)) void kernel_single_task(const Func &kernelFunc) {
```

Comment:
I would prefer you pass -sycl-std=2020 on the run command instead.

Reply:
I'm fine doing that for all three tests, but FWIW, I considered doing that an explicitly decided against it because it seemed better to have test coverage for any SYCL language mode rather than making it seem like there's something 2020-specific about the test. I'm not certain if that changes your opinion about this or not. (let me know, as I said, I'm fine either way.)

Reply:
Doesn't really change my opinion, but now I won't mind if you didn't change it either.   I think these tests may eventually get changed to use the kernel_single_task function from the mock header, as opposed to this dummied up one.

Reply:
Ohhh, I didn't realize this was an API as opposed to random test code. I see now why you asked for the changes. I'll switch the language standard mode.